app/src/services/AddonUpdater.php

```diff
@@ -358,7 +367,7 @@ private function updateAuthors(AddonVersion $version, Version $package)
                 }
 
                 if ($details->getEmail()) {
-                    $author = AddonAuthor::get()->filter('Email', $details->getEmail())->first();
+                    $author = AddonAuthor::get()->filter(Email::class, $details->getEmail())->first();
```

Comment:
This looks like an upgrader error, assuming 'Email' is a DB field on AddonAuthor